pkg/repo/validate.go

```diff
@@ -99,6 +101,7 @@ func (r *Repo) Validate() (
 		if kepParseErr != nil {
 			return warnings, valErrMap, errors.Wrap(kepParseErr, "parsing KEP file")
 		}
+		kep.Filename = filename
```

Comment:
This lets me print out an error where the offending file is actually referenced and I can fix it more easily.